pkg/vm/opcodebench_test.go

```diff
@@ -74,714 +75,1578 @@ func opParamSlotsPushVM(op opcode.Opcode, param []byte, sslot int, slotloc int,
 	}
 }
 
-func exceptParamPushVM(op opcode.Opcode, param []byte, ilen int, elen int, exception bool, items ...any) func() *VM {
+func exceptParamPushVM(op opcode.Opcode, param []byte, items ...any) func() *VM {
 	return func() *VM {
-		regVMF := opParamPushVM(op, param, items...)
-		v := regVMF()
-		if ilen != 0 {
-			eCtx := newExceptionHandlingContext(1, 2)
-			v.Context().tryStack.PushVal(eCtx)
-			for range ilen {
-				v.call(v.Context(), 0)
-			}
-		} else if elen != 0 {
-			for range elen {
-				eCtx := newExceptionHandlingContext(1, 2)
-				v.Context().tryStack.PushVal(eCtx)
-			}
-		}
-		if exception {
-			v.uncaughtException = &stackitem.Null{}
-		}
+		v := opParamPushVM(op, param, items...)()
+		v.Context().tryStack.PushVal(newExceptionHandlingContext(0, 0))
 		return v
 	}
 }
 
-func zeroSlice(l int) []byte {
-	return make([]byte, l)
+const countPoints = 8
+
+func getBigInts(minBits, maxBits, num int) []*big.Int {
+	step := uint((maxBits - minBits) / num)
+	curr := step - 1
+	res := make([]*big.Int, 0, num)
+	for range num {
+		res = append(res, new(big.Int).Lsh(big.NewInt(1), curr))
+		curr += step
+	}
+	return res
 }
 
-func ffSlice(l int) []byte {
-	var s = make([]byte, l)
-	for i := range s {
-		s[i] = 0xff
+func getArrayWithDeep(d int) []stackitem.Item {
+	var item stackitem.Item = stackitem.Null{}
+	for range d - 1 {
+		item = stackitem.NewArray([]stackitem.Item{item})
 	}
-	return s
+	return []stackitem.Item{item}
 }
 
-func maxNumber() []byte {
-	s := ffSlice(32)
-	s[0] = 0
-	return s
+func getStructWithDeep(d int) []stackitem.Item {
+	var item stackitem.Item = stackitem.Null{}
+	for range d - 1 {
+		item = stackitem.NewStruct([]stackitem.Item{item})
+	}
+	return []stackitem.Item{item}
 }
 
-func bigNumber() []byte {
-	s := maxNumber()
-	s[31] = 0
-	return s
+func getUniqueKeys(num, keySize int) [][]byte {
+	keys := make([][]byte, 0, num)
+	for i := range num {
+		key := make([]byte, keySize)
+		j := keySize - 1
+		for i > 0 {
+			key[j] = byte(i & 0xff)
+			i >>= 8
+			j--
+		}
+		keys = append(keys, key)
+	}
+	return keys
 }
 
-func bigNegNumber() []byte {
-	s := ffSlice(32)
-	s[0] = 80
-	return s
+func BenchmarkIs(b *testing.B) {
+	type testCase struct {
+		name   string
+		op     opcode.Opcode
+		params []byte
+		items  []any
```

Comment:
Looks like `testCase` is a common structure. Make it global, reuse from every benchmark. If some fields of `testCase` is not needed from a benchmark, just don't use them.

Reply:
s/testCase/benchCase